Program.java

```diff
@@ -0,0 +1,62 @@
+import java.util.Scanner;
+import java.util.regex.Pattern;
+public class Program {
+ 
+    public static void main(String[] args) {
+       Scanner in = new Scanner (System.in); 
+       System.out.print("Input a sentence: ");
+       String sentence = in.nextLine();
+       in.close();
+       StringBuffer strBuf = new StringBuffer (sentence);
+       
+       RevOrder (strBuf);
+       System.out.println("==========");
+       Words (sentence);
+       System.out.println("==========");
+       ReplaceSent (sentence);
+       System.out.println("==========");
+       GettingSent (sentence);
+       System.out.println("==========");
+       
+      
+    }
+    
+    //Reverse Order
+    static void RevOrder (StringBuffer Order) {
+    	Order.reverse();
+    	System.out.print("Reverse order: ");
```

Comment:
to print out everything in one line - System.out.println("Reverse order:" + order.toString()).
Read an article about how to use this:
https://www.programiz.com/java-programming/basic-input-output
Fix across all your code